packages/rum-react/test/e2e/with-router/routes.js

```diff
@@ -51,10 +63,13 @@ function App() {
         </li>
       </ul>
       <Suspense fallback={<div>Loading...</div>}>
-        <Switch>
```

Comment:
Is this file even required, given there is no switch functionality in the new router, I would be fine with deleting this part of the code.